hawc/apps/materialized/models.py

```diff
@@ -72,3 +87,80 @@ class FinalRiskOfBiasScore(MaterializedViewModel):
     content_type = models.ForeignKey(ContentType, null=True, on_delete=models.DO_NOTHING)
     object_id = models.IntegerField()
     content_object = GenericForeignKey("content_type", "object_id")
+
+    @classmethod
+    def get_dp_export(cls, assessment_id: int, ids: List[int], data_type: str) -> Tuple[Dict, Dict]:
```

Comment:
The `ids` parameter changes depending on data_type. Maybe it would be better to have separate functions like `get_animal_dp_export`, etc so that we don't have this ambiguous parameter. What do you think?

Reply:
A lot of this function can be moved to a private function for reuse, so I don't think it would add too many more lines of code.